### PR TITLE
Fixed defibrillators not working correctly on beds

### PR DESCRIPTION
**Pull request by FluffyGhoster**
Fixed defibrillators not working correctly on a bed/stretcher.

>I am free, you are free, everyone is free, shock!

### CODE REVIEW

**Comment by FluffyGhoster** [comment]
!bugfix

**Comment by FluffyGhoster** [comment]
!bugfix

**Comment by FluffyGhoster** [comment]
!review

**Review by alsoandanswer — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

File: code/modules/surgery/surgery.dm [hunk 1]
```diff
@@ -108,8 +108,12 @@
 		/obj/item/spell/mend_organs,
 		/obj/item/spell/modifier/mend_life,
 		/obj/item/spell/modifier/mend_synthetic,
+		/obj/item/grab,
+
+		//Defibrillator stuffs
 		/obj/item/defibrillator,
-		/obj/item/grab
+		/obj/item/shockpaddles/,
```
Comment (NonQueueingMatt) [inline code/modules/surgery/surgery.dm:115]:
```suggestion
		/obj/item/shockpaddles,
```

**Pull request merged**